base/src/main/java/com/instaclustr/cassandra/ldap/auth/SystemAuthRoles.java

```diff
@@ -8,7 +10,7 @@ public interface SystemAuthRoles {
 
     boolean roleMissing(String dn);
 
-    void createRole(String roleName, boolean superUser);
+    void createRole(String roleName, boolean superUser, Optional<String> defaultRoleMembership);
```

Comment:
It is not a good practice to pass optionals into method arguments.

Look e.g. here https://www.baeldung.com/java-optional#misuages